### DISCUSSION TITLE
Suitable usb-c board to use with a smartphone

### DISCUSSION THREAD

**Discussion by Picazsoo** [Dakkaron/Fairberry#24] [Q&A]
Hello @Dakkaron, I am wondering if you could recommend some specific usb-c board to use with the custom board. I have now everything at hand to begin making my own keyboard sans the suitable male usb-c OTG board.

Thanks for the great project. Ordering the hirose connectors and the custom boards themselves was a breeze with your detailed instructions.

**Comment by Dakkaron** [comment]
Hi @Picazsoo,

Sorry, I was on vacation and didn't see your post.

This one here should work: https://de.aliexpress.com/item/1005007180085896.html

What you need to get is a USB C OTG breakout board. You can also find them if you search for "USB C OTG soldering board".

Thanks for the kind words!

**Reply by Picazsoo** [reply]
Hi @Dakkaron , thanks a bunch for the suggestion. I managed to make it work in the meantime by ripping apart a usb-c (male) to usb-a (female) dongle that used to come with older google pixel phones. But this board is definitely even smaller and will order it instead. As a side note - out of the 5 boards ordered, only 4 seem to work correctly - could not get the last one to flash the bootloader no matter how hard I tried. But 4 out of 5 still seems pretty great considering the price.

**Reply by Dakkaron** [reply]
Good that you got it to work another way!

I probably should make it more clear in the documentation what kind of connector is needed.

A failed board is irritating. Did you try to complain with JLCPCB? Maybe they can refund it for you.

The only failures I've heard of before was badly soldered Hirose connectors, but I only heard of one batch with that issue. Maybe you got a faulty microcontroller?

**Reply by XantiaActiva** [reply]
Hi @Dakkaron,

first of all thank you for the project and thank you for sharing it with us! I got my board working, but at the moment its just working when connected to a PC. When connected to a mobile phone it does not work. So I thing the OTG thing is not working properly. Hopefully with the USB-C connector of your choice it will work on the phone, too. :) Thats what I purchased: https://www.ebay.de/itm/177392745709 --> only R1 is soldered, it does not work in the OTG mode.

Best regards

Fred

**Reply by XantiaActiva** [reply]
Unfortunately the link from @Dakkaron is no workling anymore, I get a 404 error. Any advises? Thank you!

**Reply by XantiaActiva** [reply]
I will try this one: https://www.ebay.de/itm/297795373088 
On the pictures it seems that R1, R2 and C1 are soldered. 
Edith: Again only R1 present. Its not working in OTG mode. :(

> I probably should make it more clear in the documentation what kind of connector is needed.

Thats an good idea. As the offers on Aliexpress, Ebay, etc. are changing its an good idea to have an documentation what to look for (e.g. what kind of resistors/capacitors should be found on the board) or what else we can look for. Thank you!

I just tried the Fairberry board with Q10 keyboard and had two adaptors (1x USB-C to USB-A OTG and 1x USB-A to USB-C) switched between the USB port of the phone and the USB port of the Fairberry. **It works!** Lucky me. :)

**Reply by Dakkaron** [reply]
Hi, sorry for the late response. This one should work:

https://www.amazon.de/dp/B07T97LC9L

The important thing is that it has a 5.1k resistor for the CC lines to tell the phone that USB OTG should be enabled.

**Reply by XantiaActiva** [reply]
Thank you! So now its clear for me what to look for when buying an USB-C OTG connector. BTW: I got 10 of the connectors mentioned below - if someone in Germany is interested I can help.

**Comment by XantiaActiva** [comment]
Next try:

https://de.aliexpress.com/item/1005008774450591.html

Lets hope the best. According to pictures and text its OTG this time.

<img width="334" height="284" alt="grafik" src="https://github.com/user-attachments/assets/2c832502-504d-4e59-af82-be6d1a026720" />

Best regards

**Reply by Dakkaron** [reply]
I see a resistor on there and it's got OTG in the description, so chances are good that it will work.

**Reply by XantiaActiva** [reply]
It works! :)

**Reply by Dakkaron** [reply]
Good to hear!
